apps/.eslintrc.js

```diff
@@ -25,4 +26,13 @@ module.exports = {
       },
     },
   ],
+
+  globals: {
+    __VUE_SSR_CONTEXT__: "readonly",
+  },
+
+  rules: {
+    "no-unused-vars": "off",
```

Comment:
why turn this off ? usually unused var point to error or at least hard to read (maintain) code